### DISCUSSION TITLE
Merry Christmas! 🌲

### DISCUSSION THREAD

**Discussion by kRHYME7** [HyDE-Project/HyDE#59] [Announcements]
# README

Just a heads up: there’s no such thing as stable dotfiles here (maybe except for the Nix stuff). So, don’t expect any releases to be rock solid. This release has been tested on my machine and a VM, but feel free to open issues if you encounter any problems.

Think of this release as a safety net in case the dotfiles deployment fails. If anything breaks in the future, you can always come back to this version. Just keep in mind that as dependencies change, these releases might not be useful down the line.

Happy tinkering! 

## What's Changed
* Include animations folder  by @T-Crypt in https://github.com/HyDE-Project/HyDE/pull/15
* Port waybar weather module pr from hyprdots by @boulder-roller in https://github.com/HyDE-Project/HyDE/pull/19
* fix globbing error at line 31 by @imtheabdi in https://github.com/HyDE-Project/HyDE/pull/23
* updating README to reflect the new file name by @imtheabdi in https://github.com/HyDE-Project/HyDE/pull/26
* feat: hyde config by @kRHYME7 in https://github.com/HyDE-Project/HyDE/pull/27
* fix: Fix bugs in cliphist module flags by @Anshul-007 in https://github.com/HyDE-Project/HyDE/pull/30
* fix: Fix suspend system, and change from swaylock to lockscreen.sh by @Anshul-007 in https://github.com/HyDE-Project/HyDE/pull/31
* PORT bluelight by @boulder-roller in https://github.com/HyDE-Project/HyDE/pull/21
* fix: small fix sysmonlaunch by @Anshul-007 in https://github.com/HyDE-Project/HyDE/pull/32
* feat: use proper variable and config for sysmonitor. by @kRHYME7 in https://github.com/HyDE-Project/HyDE/pull/34
* hyprlock mpris icons and some cava stuff by @kRHYME7 in https://github.com/HyDE-Project/HyDE/pull/38
* fix: Fix activation of sunset on startup by @Anshul-007 in https://github.com/HyDE-Project/HyDE/pull/36
* pokemon-colorscripts-git is optional and shouldnt be needed by @rishav12s in https://github.com/HyDE-Project/HyDE/pull/45
* fix monitor screenshot in screenshot.sh by @rishav12s in https://github.com/HyDE-Project/HyDE/pull/46
* better suspend command  by @rishav12s in https://github.com/HyDE-Project/HyDE/pull/47
* HyDE repository icon by @JoeArchV in https://github.com/HyDE-Project/HyDE/pull/50
* tooltip-format keybindhint.jsonc by @JoeArchV in https://github.com/HyDE-Project/HyDE/pull/56
* feat: reusable keybind parser by @kRHYME7 in https://github.com/HyDE-Project/HyDE/pull/52
* README.es.md by @JoeArchV in https://github.com/HyDE-Project/HyDE/pull/58

## New Contributors
* @T-Crypt made their first contribution in https://github.com/HyDE-Project/HyDE/pull/15
* @boulder-roller made their first contribution in https://github.com/HyDE-Project/HyDE/pull/19
* @imtheabdi made their first contribution in https://github.com/HyDE-Project/HyDE/pull/23
* @Anshul-007 made their first contribution in https://github.com/HyDE-Project/HyDE/pull/30
* @rishav12s made their first contribution in https://github.com/HyDE-Project/HyDE/pull/45
* @JoeArchV made their first contribution in https://github.com/HyDE-Project/HyDE/pull/50

**Full Changelog**: https://github.com/HyDE-Project/HyDE/compare/v0.1.1...v0.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/HyDE-Project/HyDE/releases/tag/'>Merry Christmas! 🌲</a>.</em>

**Comment by ZviAdzz** [comment]
Marry Christmas dude, thanks to you and all the contributors for the hard work you've put in, actually excited to check this out when i'm at home.
